### ISSUE TITLE
Heretic 2 (Windows CD) issue to install

### ISSUE THREAD

**Issue by samosakue** [legluondunet/MyLittleLutrisScripts#28]
There no "h2_setup.exe" instead there is a "setup.exe" when I try to install it.

**Comment by legluondunet** [comment]
What CD version do you own?
Do you have audio track on CD?

**Comment by samosakue** [comment]
I have the american version and my version has audio tracks.

**Comment by samosakue** [comment]
> Hello @samosakue, thank you for your report.
> What CD version do you own? I wrote my script with the European version.
> Do you have audio track on your H2 CD?
I have the american version and my version has audio tracks.

**Comment by samosakue** [comment]
> Hello @samosakue, thank you for your report.
> What CD version do you own? I wrote my script with the European version.
> Do you have audio track on your H2 CD?

I have the american version and my version has audio tracks.

**Comment by legluondunet** [comment]
If this CD would be available online, I can make some tests.
If you know where I could obtain it, this is my mail: legluondunet@gmail.com.

**Comment by legluondunet** [comment]
As you can see, in the root of my CD there is no setup.exe file but h2_setup.exe:
![Capture d’écran de 2020-03-28 23-32-33](https://user-images.githubusercontent.com/325405/77835411-8faaa580-714c-11ea-83e4-a345c1504913.png)


However there is a setup.exe file in the setup folder.

**Comment by samosakue** [comment]
How can I do that? Do you want me to copy the content of the cd and email
it to you?

**Comment by samosakue** [comment]
How can I do that? Do you want me to copy the content of the cd and email
it to you

**Comment by legluondunet** [comment]
There is another way, could you type this command in a terminal:

$ find /media/your_profile_name/HERETIC_II/ > disque.txt

or whereever your disc is mounted and post the disque.txt here.

**Comment by samosakue** [comment]
[disque.txt](https://github.com/legluondunet/MyLittleLutrisScripts/files/4399524/disque.txt)

**Comment by samosakue** [comment]
Here it is.
[disque.txt](https://github.com/legluondunet/MyLittleLutrisScripts/files/4399528/disque.txt)

**Comment by samosakue** [comment]
Why you say different version? Are you able to fix this issue?

**Comment by legluondunet** [comment]
@samosakue could you tell me more about your version, it doesn't look an official activision version.

**Comment by samosakue** [comment]
> @samosakue could you tell me more about your version, it doesn't look an official activision version.

It is an an official american version.

**Comment by samosakue** [comment]
> @samosakue could you tell me more about your version, it doesn't look an official activision version.

Any update. I told you that it is an official activision version.

**Comment by legluondunet** [comment]
@samosakue This version is very different from the one I own, I need time to write a new script, please be patient.

**Comment by samosakue** [comment]
@legluondunet ok. Just keep me up to date on any progress. Thanks

**Comment by legluondunet** [comment]
Hello @samosakue , I'm sorry but I will not work on that script for several reasons:
- I write only scripts for games I own, I don't own this version, I only own EU version.
- I'm working in parallel on a lot of projects and I have no time to work on it
If you are motivated, you can use my Heretic 2 script to write one for your version.
On this page, https://lutris.net/games/heretic-ii/, click on fork installer on my "Wine CD + DxWnD + WS patch version". If you have questions you can obtain help here or better with the Lutris community on Discord: https://discordapp.com/invite/Pnt5CuY.

**Comment by legluondunet** [comment]
Close as won't fix.

**Comment by samosakue** [comment]
Can I send you a copy of the game. I really want to play this game so bad.
I'm in America.

**Comment by legluondunet** [comment]
Could you find another version? an European version with your native language (English?)

**Comment by samosakue** [comment]
Why not the american version? It should not be fair for us Americans to not
be able to play this game on Linux.

**Comment by legluondunet** [comment]
Hello @samosakue,  just created another script for Heretic II Windows CD launching setup.exe.
Enjoy!
